package.json

```diff
@@ -87,6 +87,7 @@
     "topics": {
       "schema": {
         "external": true,
+        "description": "Generate metadata files."
```

Comment:
```suggestion
        "description": "Generate metadata files.",
```

Reply:
This should fix the pjson.

UT didn't run because of invalid json: https://github.com/salesforcecli/plugin-sobject/actions/runs/4138936348/jobs/7155948892#step:3:19